tools/cpp/windows_cc_configure.bzl

```diff
@@ -338,17 +395,32 @@ def configure_windows_toolchain(repository_ctx):
     escaped_tmp_dir = escape_string(
         get_env_var(repository_ctx, "TMP", "C:\\Windows\\Temp").replace("\\", "\\\\"),
     )
-    msvc_cl_path = find_msvc_tool(repository_ctx, vc_path, "cl.exe").replace("\\", "/")
+
+    llvm_path = ""
+    if _use_clang_cl(repository_ctx):
+        llvm_path = find_llvm_path(repository_ctx)
+        if not llvm_path:
+            auto_configure_fail("\nUSE_CLANG_CL is set to 1, but Bazel cannot find Clang installation on your system.\n"
+                                + "Please install Clang via http://releases.llvm.org/download.html\n")
```

Comment:
The url looks quite simple and stable, I think it's fine to suggest this one. If it does go stale, I think it will probably redirect the user to a new page somehow.